dll/win32/browseui/settings.h

```diff
@@ -12,12 +12,13 @@
 
 struct ShellSettings
 {
-    BOOL fLocked = FALSE;
-    BOOL fShowGoButton = FALSE;
-    BOOL fStatusBarVisible = FALSE;
+    BOOL fLocked = TRUE;
+    BOOL fShowGoButton = TRUE;
+    BOOL fStatusBarVisible = TRUE;
 
     void Save();
     void Load();
+    void Reset() { ShellSettings().Save(); }
```

Comment:
Weird that when resetting you would save the settings. Shouldn't instead load either the default ones or existing ones from the registry?

Reply:
> Shouldn't instead load either the default ones or existing ones from the registry?

It creates a fresh instance with the defaults set `ShellSettings()` and saves that `.Save()`

Reply:
OK that's perfect.